src/docker-utils.coffee

```diff
@@ -206,6 +274,6 @@ do ->
 						console.log 'Progress error: Unknown layer ' + id + ' downloaded by docker. Progress not correct.'
 						totalSize = null
 				downloadedSize = completedSize + currentSize
-				percentage = calculatePercentage(downloadedSize, totalSize)
+				percent = calculatePercentage(downloadedSize, totalSize)
 
-				onProgress({ downloadedSize, totalSize, percentage })
+				onProgress({ downloadedSize, totalSize, percent })
```

Comment:
Can you keep this as `percentage` to match up with https://github.com/resin-io/docker-progress - it came from this code originally (but has since had push progress added) and is what we use in the builder, it'd be nice to keep the api the same between them